### PR TITLE
Tutorial Update - `inheritFrom`

### PR DESCRIPTION
**Pull request by 96LawDawg**
Fixes broken label sizes; adds new examples.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1750/pr-test (or any other room on that server)

After merging, a backup will be available at `https://beta.virtualtabletop.io/editor/PR1750-pr-test`.

File: library/tutorials/inheritFrom/0.json [hunk 1]
```diff
@@ -820,9 +829,10 @@
             "y": 0,
             "width": 103,
             "height": 160,
-            "valueType": "static",
-            "value": "/i/cards-default/2B.svg",
-            "color": "transparent"
+            "color": "transparent",
+            "dynamicProperties": {
+              "value": "/i/cards-default/2B.svg"
```
Comment (ArnoldSmith86) [inline library/tutorials/inheritFrom/0.json:834]: How did this happen? It was static but now it is dynamic and it doesn't look like a property name. Was this the file updater or a manual change?

File: library/tutorials/inheritFrom/0.json [hunk 2]
```diff
@@ -1249,5 +1256,131 @@
...
+    "text": "Inherits angle and value",
+    "height": 60,
+    "parent": "i6gw"
+  },
+  "dbeyh2": {
+    "type": "label",
+    "id": "dbeyh2",
+    "x": 5,
+    "z": 145,
+    "text": "does not inherits css",
```
Comment (ArnoldSmith86) [inline library/tutorials/inheritFrom/0.json:1274]: Typo.

File: library/tutorials/inheritFrom/0.json (hunk 1 above)
Comment (96LawDawg) [reply library/tutorials/inheritFrom/0.json]: No idea.  It isn't that way in the file that Raphael linked.  I just downloaded that and uploaded it to Git.  Don't think I touched anything.  Has to have been a file updater.

Comment (RaphaelAlvez) [reply library/tutorials/inheritFrom/0.json]: File updater. I don't recall changing anything on cards

**Comment by RaphaelAlvez** [comment]
Thanks for fixing it @96LawDawg.

If nothing else is required, i have no problem merging this. The css tutorial should come with the spinner tutorial and I have no idea for how long it will take

**Review by ArnoldSmith86 — APPROVED** [review]

**Pull request merged**